modules/adding-bare-metal-host-to-cluster-using-web-console.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+[id="adding-bare-metal-host-to-cluster-using-web-console_{context}"]
+= Adding a bare metal host to the cluster using the web console
+
+You can add bare metal hosts to the cluster in the web console.
+
+.Prerequisites
+
+* Install an {op-system} cluster on bare metal.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the web console, navigate to *Compute* -> *Bare Metal Hosts*.
+. Select *Add Host* -> *New with Dialog*.
+. Specify a unique name for the new bare metal host.
+. Set the boot MAC address.
+. Optionally, enable power management for the host. This allows {product-title} to control the power state of the host.
+. Enter the user credentials for the host's baseboard management controller (BMC).
```

Comment:
before entering user credentials for BMC, we also need add a step setting `Baseboard Management Console(BMC) Address`